api/server/init.go

```diff
@@ -56,6 +58,20 @@ func getEnvInt(key string, fallback int) int {
 	return fallback
 }
 
+func getEnvDuration(key string, fallback time.Duration) time.Duration {
+	var err error
+	res := fallback
+	if tmp := os.Getenv(key); tmp != "" {
+		res, err = time.ParseDuration(tmp + "s")
+		if err != nil {
+			res = fallback
```

Comment:
setting res to fallback right before panic() is slightly odd. We can remove this line?

Reply:
true! I wrongly added it in two places